FRAuth/FRAuth/SocialLogin/AppleSignInUserStructs.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  AppleSignInHandler.swift
+//  FRAuth
+//
+//  Copyright (c) 2022 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+import AuthenticationServices
+
+/*
+ Apple Sign in example JSON response for 3rd party applications.
+ This format is expected to be passed to AM's IdPCallback
+ {
+     "code": "1234781237213123...",
+     "id_token": "PEHANCKakenAKDNFAhekqoioenakdfnn53i2kKK23k2k2nn3...",
+     "state": "..."
+     "user": {
+         "name": {
+             "firstName": "Kurt",
+             "lastName": "Cobain"
+         },
+         "email": "kurt@example.com"
+     }
+ }
+         
+ */
```

Comment:
Do we want test for this class ? i am not usually super worried if its just a getter and setter , but this has some logic for code , id_token, email

Reply:
yeap I think some tests around this would be good. I am working to add some stuff on testing